docs/resources/Application.md

```diff
@@ -16,6 +16,7 @@
 | terms_of_service_url?  | string                                                                     | the url of the app's terms of service                                                                                     |
 | privacy_policy_url?    | string                                                                     | the url of the app's privacy policy                                                                                       |
 | owner?                 | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | partial user object containing info on the owner of the application                                                       |
+| summary *(deprecated)* | string                                                                     | **deprecated and will be removed in v11.** An empty string.                                                               |
```

Comment:
IIRC this was removed previously to reflect the deprecation?

Reply:
Yeah it was, but I added it back in since the empty string still appears in the payload. Even if deprecated, if a key is still present it feels like it should still be included